evm/src/all_stark.rs

```diff
@@ -146,7 +180,30 @@ mod tests {
         keccak_keccak_input_output.extend(Column::singles(
             (0..2 * NUM_INPUTS).map(keccak::registers::reg_output_limb),
         ));
-        let cross_table_lookups = vec![CrossTableLookup::new(
+
+        let cpu_memory_values: Vec<Vec<_>> = (0..4)
+            .map(|op| {
+                let mut vals = vec![
+                    cpu::columns::MEMORY_TIMESTAMP,
```

Comment:
I think our plan was to pass something like `NUM_MEMORY_OPS * clock + op` here, so the `NUM_MEMORY_OPS` ops occur in a specific sequence. Maybe add a TODO to do that later?

The CTL code doesn't support linear combinations like that currently, but maybe it wouldn't be too hard to add since they're supported for filters. cc @wborgeaud

Reply:
Ohh I could try to work on that! I'd like to get my hands dirty with CTL stuff a little

Reply:
Hm normally the linear combinations supported are the same for filters and columns now. Here `NUM_MEMORY_OPS` and `op` are constants and `clock` is a column, right? If so you can use `Column::linear_combination_with_constant([(clock, NUM_MEMORY_OPS)], ops)`.

Reply:
Nice, thanks!!